### PR TITLE
GWAS++ Test plan

### PR DESCRIPTION
**Pull request by atharvar28**
[GQE-27](https://ctds-planx.atlassian.net/browse/GQE-27)

[GQE-27]: https://ctds-planx.atlassian.net/browse/GQE-27?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/testplans/GWAS/GWAS++_test_plan.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+## Technical Documentation
+
+The documentation about the GWAS app can be found in QA and PreProd
+
+[Documentation from QA](https://qa-mickey.planx-pla.net/dashboard/Public/documentation/index.html#gen3-gwas)
+
+## Scope
+
+GWAS++ provides unified process tool for the scientists convert input files/folders into input manifest for mariner
```
Comment (vzpgb) [inline docs/testplans/GWAS/GWAS++_test_plan.md:15]:
```suggestion
Gen3 GWAS App provides the interface to perform high throughput GWAS on Million Veteran Program (MVP) data using the University of Washington Genesis pipeline.
```

File: docs/testplans/GWAS/GWAS++_test_plan.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+-   Select Outcome Phenotype (Continuous or Dichotomous). Select variables.Click Submit. See the change in Attrition Table. Click Submit
+-   Select Covariate Phenotype (Continuous or Dichotomous). Select covariate variables. Click Add. See the change in Attrition Table.
+-   Configure GWAS.
+-   Submitting the job.
+-   Checking if the navigation and deletion button work as expected.
+
+## Test Scenarios
+
+-   Testing whole workflow to submit a GWAS job with both continuous and dichotomous phenotype and covariate phenotypes.
+-   Negative tests - unauthorized permissions to GWAS, Selecting non-phenotype file at intial stages of the workflow
```
Comment (vzpgb) [inline docs/testplans/GWAS/GWAS++_test_plan.md:31]: "Selecting non-phenotype file" - users do not select the phenotype file, they access the cohorts

File: docs/testplans/GWAS/GWAS++_test_plan.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+## Scope
+
+GWAS++ provides unified process tool for the scientists convert input files/folders into input manifest for mariner
+
+## Workflow
+
+-   User logs in and navigates to Apps on commons homepage
+-   Selects GWAS++ App
+-   Selects the Cohort and sees change in Attrition table on the top for selected cohort. Click Next
```
Comment (vzpgb) [inline docs/testplans/GWAS/GWAS++_test_plan.md:21]: User can add the cohort by clicking "Add new cohort".

**Review by vzpgb — COMMENTED** [review]
Looks good, thanks! I added several minor comments

**Review by vzpgb — APPROVED** [review]

**Review by vzpgb — APPROVED** [review]

**Review by haraprasadj — APPROVED** [review]

**Pull request merged**